cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -304,11 +312,11 @@ instance HasSeverityAnnotation (WithMuxBearer peer MuxTrace) where
     MuxTraceRecvDeltaQSample {} -> Debug
     MuxTraceSDUReadTimeoutException -> Notice
     MuxTraceSDUWriteTimeoutException -> Notice
-    MuxTraceStartEagerly _ _ -> Debug
-    MuxTraceStartOnDemand _ _ -> Debug
-    MuxTraceStartedOnDemand _ _ -> Debug
-    MuxTraceTerminating {} -> Debug
+    MuxTraceStartEagerly _ _ -> Info
+    MuxTraceStartOnDemand _ _ -> Info
+    MuxTraceStartedOnDemand _ _ -> Info
     MuxTraceShutdown -> Debug
```

Comment:
This doesn't match the commit message.

Reply:
Done; I fixed this after pushing, but I didn't wanted intervene the CI.